### DISCUSSION TITLE
windows下，下载alist-windows-4.0-amd64.zip ，杀软avast报毒，直接被删除了。

### DISCUSSION THREAD

**Discussion by zjxndsky** [AlistGo/alist#1835] [General]
windows下，下载alist-windows-4.0-amd64.zip ，杀软avast报毒，直接被删除了。而且访问https://alist.nn.ci/，也被avast阻止，因为它感梁了RUL:Phishing. 请问怎么破。

**Comment by ihxjie** [comment]
恢复并加入白名单即可
如果担心有病毒的话，可以下载源代码查看并自行编译。
